src/components/hv-date-field/index.tsx

```diff
@@ -54,20 +59,19 @@ export default class HvDateField extends PureComponent<HvComponentProps> {
 
   /**
    * Given a ISO date string (YYYY-MM-DD), returns a Date object. If the string
-   * cannot be parsed or is falsey, returns null.
+   * cannot be parsed or is falsey, returns default new Date() value.
    */
-  static createDateFromString = (
-    value?: string | null,
-  ): Date | null | undefined => {
+  static createDateFromString = (value?: string | null): Date => {
     if (!value) {
-      return null;
+      return new Date();
```

Comment:
that is changing the behavior, we'll have to verify on the final PR if that's needed